backend/dal/Services/SecurityDepositReturnService.cs

```diff
@@ -35,6 +35,12 @@ public PimsLease UpdateLeaseDepositReturn(long leaseId, long leaseRowVersion, Pi
         {
             _user.ThrowIfNotAuthorized(Permissions.LeaseEdit);
             ValidateServiceCall(leaseId, leaseRowVersion);
+            var currentHolder = _securityDepositReturnRepository.GetById(deposit.SecurityDepositReturnId).PimsSecurityDepositReturnHolder;
+            if (currentHolder != null)
+            {
+                deposit.PimsSecurityDepositReturnHolder.SecurityDepositReturnHolderId = currentHolder.SecurityDepositReturnHolderId;
```

Comment:
What is the purpose of this logic, is this to prevent someone from inadvertently updating the holder?

Reply:
This is needed since the front does not have the holder table id. In the front end, the contact is selected (contact view Id) wherein the backend that needs to be set to the existing holder id